Client/Frontend/Intro/IntroScreenWelcomeViewV2.swift

```diff
@@ -0,0 +1,281 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+import Shared
+import SnapKit
+
+/* The layout for update view controller.
+
+[Top View] and [Stack View] are put together in another
+uiview called combined view mainly to put the whole thing
+in the middle of the screen.
+ 
+|----------------|
+|           Cross| Cross button is on top right corner
+|                |
+|----------------|----------[Combined View]--------------
+|                |
+|     Image      | [Top View]
+|                |      -- Has title image view
+|Title Multiline |      -- Title label view
+|----------------|
+|                | [Stack View] - Fixed height and
+|                |  contains subviews with title and description
+|     Title      |  -- automaticPrivacyView
+|   Description  |      -- Title & Description label uiviews
+|                |
+|     Title      |  -- fastSearchView
+|   Description  |      -- Title & Description label uiviews
+|                |
+|     Title      |  -- safeSyncView
+|   Description  |      -- Title & Description label uiviews
+|                |
+|----------------|----------[Combined View]--------------
+|                |
+|    [Next]      | Bottom View - Only Has next button
+|----------------|
+
+*/
+
+class IntroScreenWelcomeViewV2: UIView {
+    // Private vars
+    private var fxTextThemeColour: UIColor {
+        // For dark theme we want to show light colours and for light we want to show dark colours
+        return UpdateViewController.theme == .dark ? .white : .black
+    }
+    private var fxBackgroundThemeColour: UIColor {
+        return UpdateViewController.theme == .dark ? UIColor(colorString: "242327") : .white
+    }
+    private lazy var titleImageView: UIImageView = {
+        let imgView = UIImageView(image: #imageLiteral(resourceName: "splash"))
+        imgView.contentMode = .scaleAspectFit
+        imgView.clipsToBounds = true
+        return imgView
+    }()
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = Strings.CardTitleWelcome
+        label.textColor = fxTextThemeColour
+        label.font = UIFont.systemFont(ofSize: 26, weight: .semibold)
```

Comment:
Should we be using DynamicFontSize?

Reply:
Hmm I asked UX and tried on all devies. It looked fine with font 26. Think we can keep it the way it is for now. Thoughts? 🤔 

![image](https://user-images.githubusercontent.com/8919439/81970022-dfeea180-95ec-11ea-8ace-9cbb3366c1ca.png)

Reply:
^ This was approved by the UX team

Reply:
I should have been clearer, isn't DynamicFontSize what we use for font accessibility support? I am never clear on when we are supposed to use that or not.

Reply:
Not really sure about font accessibility support :(